File: 151002/Redkovskiy/StoryTeller/discussion-service/src/main/java/org/example/discussionservice/model/Post.java
package org.example.discussionservice.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.cassandra.core.mapping.PrimaryKey;
import org.springframework.data.cassandra.core.mapping.Table;

@Table("tbl_post")
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Post {

    @PrimaryKey
    private PostId postId;

    private String content;

}

File: 251003/Pankratiev/rest/discussion/src/main/kotlin/bsuir/dc/discussion/entity/Post.kt
package bsuir.dc.discussion.entity

import jakarta.validation.constraints.Size
import org.springframework.data.cassandra.core.mapping.PrimaryKey
import org.springframework.data.cassandra.core.mapping.Table

@Table("tbl_post")
data class Post(
    @PrimaryKey
    val key: PostKey,

    val issueId: Long,

    @field:Size(min = 2, max = 2048)
    val content: String
)
